common/src/main/java/org/apache/drill/common/logical/FormatPluginConfigBase.java

```diff
@@ -27,11 +27,20 @@
   static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(FormatPluginConfigBase.class);
 
 
-  public synchronized static Class<?>[] getSubTypes(DrillConfig config){
-    List<String> packages = config.getStringList(CommonConstants.STORAGE_PLUGIN_CONFIG_SCAN_PACKAGES);
-    Class<?>[] sec = PathScanner.scanForImplementationsArr(FormatPluginConfig.class, packages);
-    logger.debug("Adding Format Plugin Configs including {}", (Object) sec );
-    return sec;
+  public synchronized static Class<?>[] getSubTypes(DrillConfig config) {
+    List<String> packages =
+        config.getStringList(CommonConstants.STORAGE_PLUGIN_CONFIG_SCAN_PACKAGES);
+    Class<?>[] pluginClasses =
+        PathScanner.scanForImplementationsArr(FormatPluginConfig.class, packages);
+    if (logger.isDebugEnabled()) {
+      final StringBuilder sb = new StringBuilder();
```

Comment:
There believe there are at least packages in common that claim to hold general utilities. If you can't find a class to put in a utility function, I think it would be fine to create one.